packages/crml-cennzx-spot/package.json

```diff
@@ -19,12 +19,12 @@
   },
   "dependencies": {
     "@cennznet/api": "^0.20.7",
-    "@cennznet/crml-generic-asset": "^0.20.7",
+    "@cennznet/crml-generic-asset": "^1.0.0-alpha.0",
     "@cennznet/types": "^0.20.7",
```

Comment:
@AlexZhenWang I suppose the script bump version for `@cennznet/types`, do you understand why it did not?

Reply:
Yeah, needs fixing

Reply:
But it works for other packages? What script did you use?

Reply:
It was suggested by using [plugnet-dev-version](https://github.com/plugblockchain/dev.js/blob/master/packages/dev/scripts/plugnet-dev-version.sh); As we are removing plugnet, I ran the script directly like ` yarn run lerna version 1.0.0-alpha.0 --preid alpha --yes --no-git-tag-version --no-push`

Reply:
ah, thanks Ben, let me have a look